code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2094,6 +2094,40 @@
 	A.material_flags = applied_material_flags
 	A.set_custom_materials(metal_dat)
 
+/datum/reagent/obamium
+	name = "Obamium"
+	data = list("material"=null)
+	description = "A brown liquid made from a previous president. Can turn anything into his liking"
+	color = "#341c02"
+	taste_mult = 100 //god the flavor
+	taste_description = "healthcare and drone-strikes" //what a legend
```

Comment:
should "drone-strikes" have a hyphen in it?

Reply:
i dont know im not fluent in american

Reply:
should be drone strikes, not drone-strikes

Reply:
```suggestion
	taste_description = "healthcare and drone strikes" //what a legend
```